src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -381,6 +381,9 @@
   <data name="CmdFocusDesc" xml:space="preserve">
     <value>Launch the window in focus mode</value>
   </data>
+  <data name="CmdSavedLayoutArgDesc" xml:space="preserve">
+    <value>Specify a saved window layout idx to initially load layout from. </value>

```

Comment:
I have a general question about this. I did add something to the command line, but it isn't really meant to be publicized since I can only guarantee that it works properly programmatically or by looking at the state file.

Reply:
I updated the text to spell out `index` and also mention how it will be used (that it takes from `state.json`)